src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx

```diff
@@ -0,0 +1,59 @@
+import { CardSection } from 'components/shared/CardSection'
+import FundingCycleDetailsCard from 'components/shared/Project/FundingCycleDetailsCard'
+import { LoadingOutlined } from '@ant-design/icons'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import { useContext } from 'react'
+import { V2FundingCycleRiskCount } from 'utils/v2/fundingCycle'
+
+import FundingCycleDetails from './FundingCycleDetails'
+import PayoutSplitsCard from './PayoutSplitsCard'
+import ReservedTokensSplitsCard from './ReservedTokensSplitsCard'
+
+export default function UpcomingFundingCycle({
+  expandCard,
+}: {
+  expandCard?: boolean
+}) {
+  const {
+    queuedFundingCycle,
```

Comment:
this is fine for now but I think we don't need to have all of this queued stuff in project context. We can load it only when necessary to save some Infura burden.

Reply:
Ah yeah true wasn't considering that at all. Will defs be worth removing eventually